src/main/java/uk/gov/hmcts/darts/common/exceptions/soap/documentum/ServiceExceptionType.java

```diff
@@ -55,7 +55,7 @@ public ServiceExceptionType(String code, Throwable cause, String arg) {
 
         // add one exception
         DfsExceptionHolder holder = new DfsExceptionHolder();
-        holder.setExceptionClass(ServiceExceptionType.class.getCanonicalName());
+        holder.setExceptionClass("com.emc.documentum.fs.rt.ServiceContextLookupException");
```

Comment:
Do we need to consider the other types of exception class that could occur as part of this PR?  This will fix the specific scenario that we are looking at but I assume that this same class is used when other documentum exceptions are thrown

Reply:
yeah this fits with the documentum code so should be fine. Lets try it.